### PR TITLE
Remove references to plans

### LINKED ISSUE #3389: Vendor: Remove references to plans & resources
### Where on docs.platform.sh should be changed?

Throughout the docs

### What exactly should be updated?

- Remove references to plans. See spreadsheet in context.

### Additional context

- DG2 & DG3 section will be removed in #3374.
- Language issues & larger story changes post-vendorization to be addressed in future PRs.

### PR DESCRIPTION
**Pull request by AnouckColson**
## Why

Closes #3389 

## What's changed

- Removed version tables on service & language pages
- Removed references to Dedicated infrastructures & Grid

### CODE REVIEW

File: sites/friday/src/increase-observability/integrate-observability/blackfire.md [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+1. Retrieve [startup errors](#1-retrieve-startup-errors).
+2. Retrieve your [Blackfire logs](#2-retrieve-your-blackfire-logs).
+3. Send this data to [Blackfire Support](https://support.blackfire.io).
+
+### 1. Retrieve startup errors
+
+To retrieve startup errors, run the following command:
+
+```bash
+platform ssh -- php -d display_startup_errors=on --ri blackfire
```
Comment (thomasdiluccio) [inline sites/friday/src/increase-observability/integrate-observability/blackfire.md:148]: We need to "vendorize" this CLI command

File: sites/friday/src/increase-observability/integrate-observability/blackfire.md [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+```
+
+### 2. Retrieve your Blackfire logs
+
+To retrieve your Blackfire logs, follow these steps:
+
+1.  On the environment where you're facing issues, create the following [variable](../../development/variables/set-variables.md):
+
+    ```bash
+    platform variable:create php:blackfire.log_file --value /tmp/blackfire.log
```
Comment (thomasdiluccio) [inline sites/friday/src/increase-observability/integrate-observability/blackfire.md:158]: same vendorization needed

File: sites/friday/src/increase-observability/integrate-observability/blackfire.md [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+    ```
+
+2.  To set the verbosity of the logs to level 4 (debug level), create the following variable:
+
+    ```bash
+    platform variable:create php:blackfire.log_level --value 4
```
Comment (thomasdiluccio) [inline sites/friday/src/increase-observability/integrate-observability/blackfire.md:164]: same vendorization needed

File: sites/friday/src/increase-observability/integrate-observability/blackfire.md [hunk 4]
```diff
@@ -0,0 +1,181 @@
...
+    ```
+
+3.  Start a profile or build.
+
+4.  To display the logs, run the following command:
+
+    ```bash
+    platform ssh -- cat /tmp/blackfire.log > blackfire.log
```
Comment (thomasdiluccio) [inline sites/friday/src/increase-observability/integrate-observability/blackfire.md:172]: same vendorization needed

File: sites/friday/src/increase-observability/integrate-observability/blackfire.md [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+    ```
+
+After you've retrieved the logs, you can disable them.
+To do so, run the following commands:
+
+```bash
+platform variable:delete php:blackfire.log_file
+platform variable:delete php:blackfire.log_level
```
Comment (thomasdiluccio) [inline sites/friday/src/increase-observability/integrate-observability/blackfire.md:180]: same vendorization needed

File: sites/friday/src/increase-observability/logs/access-logs.md [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+<--->
+
++++
+title=Using the CLI
++++
+
+1. Get a list of activities by running
+
+   ``` bash
+   platform activity:list -e {{% variable "ENVIRONMENT_NAME" %}}
```
Comment (thomasdiluccio) [inline sites/friday/src/increase-observability/logs/access-logs.md:35]: same vendorization needed

File: sites/friday/src/increase-observability/logs/access-logs.md [hunk 7]
```diff
@@ -0,0 +1,152 @@
...
+   ```
+
+   Pass the `--start` flag to get activities from a specific date in the past.
+
+2. To see details about the activity's state and timing, run
+
+   ``` bash
+   platform activity:get {{% variable "ACTIVITY_ID" %}}
```
Comment (thomasdiluccio) [inline sites/friday/src/increase-observability/logs/access-logs.md:43]: same vendorization needed

File: sites/friday/src/increase-observability/logs/access-logs.md [hunk 8]
```diff
@@ -0,0 +1,152 @@
...
+   ```
+
+   Where {{% variable "ACTIVITY_ID" %}} comes from the list in step 1.
+
+3. Get a log of any given activity by running
+
+   ``` bash
+   platform activity:log {{% variable "ACTIVITY_ID" %}}
```
Comment (thomasdiluccio) [inline sites/friday/src/increase-observability/logs/access-logs.md:51]: same vendorization needed

File: sites/friday/src/increase-observability/logs/access-logs.md [hunk 9]
```diff
@@ -0,0 +1,152 @@
...
+
++++
+title=Using the CLI
++++
+
+Use the `platform log` command and specify the type of log you want.
+For example, to get the access log, run:
+
+```bash
+platform log -e {{% variable "ENVIRONMENT_NAME" %}} access
```
Comment (thomasdiluccio) [inline sites/friday/src/increase-observability/logs/access-logs.md:90]: same vendorization needed

File: sites/friday/src/increase-observability/logs/access-logs.md [hunk 10]
```diff
@@ -0,0 +1,152 @@
...
+<--->
+
++++
+title=Using SSH directly
++++
+
+1. Access the container by running 
+
+   ``` bash
+   platform ssh -e {{% variable "ENVIRONMENT_NAME" %}}
```
Comment (thomasdiluccio) [inline sites/friday/src/increase-observability/logs/access-logs.md:105]: same vendorization needed

File: sites/friday/src/increase-observability/logs/access-logs.md [hunk 11]
```diff
@@ -0,0 +1,152 @@
...
+| `php.access`  | No             | A record of all requests to the PHP service. See [PHP access record format](#php-access-record-format). |
+| `post_deploy` | No             | The output of the [`post_deploy` hook](../../create-apps/hooks/hooks-comparison.md#post-deploy-hook). Only exists after a `post_deploy` hook has run. |
+
+#### PHP access record format
+
+The formatting of `php.access.log` is determined by the PHP settings.
+To determine the format, run the following:
+
+```bash
+platform ssh cat -n /etc/php/{{< variable "PHP_VERSION" >}}-zts/fpm/php-fpm.conf | grep "access.format"
```
Comment (thomasdiluccio) [inline sites/friday/src/increase-observability/logs/access-logs.md:138]: same vendorization needed

**Review by thomasdiluccio — DISMISSED** [review]
In OK with the content.

On a side note, there are still many occurrences of the "platform" CLI. Should we want to update those in that PR or in a specific other PR?

**Comment by AnouckColson** [comment]
@thomasdiluccio Thanks for the review and approval, we've got https://github.com/platformsh/platformsh-docs/pull/3398 to address all the CLI prefix occurrences ;)

**Review by chadwcarlson — APPROVED** [review]

**Pull request merged**